src/accelerate/accelerator.py

```diff
@@ -2031,6 +2041,8 @@ def backward(self, loss, **kwargs):
             return
         elif self.scaler is not None:
             self.scaler.scale(loss).backward(**kwargs)
+        elif learning_rate is not None and self._has_lomo_optimizer:
+            self._lomo_backward(loss, learning_rate)
```

Comment:
Does this need to interact with `self.scaler` for AMP? That may be important!

Reply:
hmm not sure here actually 🤯 I need to investigate a bit ..